### ISSUE TITLE
Creating conda environment

### ISSUE THREAD

**Issue by nmtate** [nmtate/Heritability_GeneArch#5]

**Comment by nmtate** [comment]
Created conda environment for the equine genomic prediction project:

1. Created a new conda environment with Python 3.11:
conda create -n equine-gs python=3.11 -y

2. Activated the environment:
conda activate equine-gs

3. Installed required packages using conda (not pip), all from conda-forge:
conda install -c conda-forge -c bioconda python=3.11 numpy pandas scipy scikit-learn xgboost joblib shap tqdm pyarrow numexpr r-base r-essentials r-optparse r-data.table r-glmnet r-rrblup r-sommer r-aghmatrix r-bglr r-kernlab r-irlba r-bigstatsr r-bigsnpr plink plink2 gcta bcftools htslib -y

Environment completed successfully. To reactivate later:
conda activate equine-gs

**Comment by nmtate** [comment]
### Python Packages

| Package        | Purpose                                      |
| -------------- | -------------------------------------------- |
| `python=3.11`  | Interpreter version                          |
| `numpy`        | Arrays / linear algebra                      |
| `pandas`       | Data manipulation                            |
| `scipy`        | Kernel Ridge Regression (Python RKHS option) |
| `scikit-learn` | Elastic Net, LASSO, Kernel Ridge, metrics    |
| `xgboost`      | Gradient boosting genomic prediction         |
| `joblib`       | Parallelization                              |
| `shap`         | Feature importance (optional but useful)     |
| `tqdm`         | Progress bars                                |
| `pyarrow`      | Fast Parquet/Feather IO                      |
| `numexpr`      | Fast matrix math                             |

**Comment by nmtate** [comment]
### R Packages

| Package        | Function / Role in Pipeline                                     |
| -------------- | --------------------------------------------------------------- |
| `r-base`       | R language runtime (core R)                                     |
| `r-essentials` | Installs core tidyverse packages and IRkernel for Jupyter       |
| `r-optparse`   | Command-line argument parsing in R scripts                      |
| `r-data.table` | Fast and memory-efficient data handling                         |
| `r-glmnet`     | Elastic Net and LASSO regression                                |
| `r-rrblup`     | GBLUP / RR-BLUP / GWAS (classic animal breeding models)         |
| `r-sommer`     | GBLUP, multi-kernel models, mixed models for genomic prediction |
| `r-aghmatrix`  | Builds A/G relationship matrices (GRMs)                         |
| `r-bglr`       | Bayesian genomic prediction, including RKHS                     |
| `r-kernlab`    | Kernel methods including RKHS / Gaussian RBF                    |
| `r-irlba`      | Fast PCA/SVD for large genomic matrices                         |
| `r-bigstatsr`  | Memory-efficient big matrix operations (optional)               |
| `r-bigsnpr`    | Memory-efficient SNP storage & processing (optional)            |

**Comment by nmtate** [comment]
### Genetic Tools

| Tool       | Purpose                            |
| ---------- | ---------------------------------- |
| `plink`    | SNP filtering                      |
| `plink2`   | Modern PLINK v2 (fast)             |
| `gcta`     | GRMs & REML heritability           |
| `bcftools` | Manipulate VCFs / BGZIP / indexing |
| `htslib`   | Underlying support library         |
